weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage/CallSummary.tsx

```diff
@@ -34,6 +35,14 @@ export const CallSummary: React.FC<{
   );
   const costData = call.traceCall?.summary?.weave?.costs;
 
+  const inputBytes = call.traceCall?.inputs
+    ? JSON.stringify(call.traceCall?.inputs).length
+    : 0;
+  const outputBytes = call.traceCall?.output
+    ? JSON.stringify(call.traceCall?.output).length
+    : 0;
+  const totalBytesStored = inputBytes + outputBytes;
+
```

Comment:
Used ternary to catch edge cases because `null` will return 4 bytes, and `undefined` will error out. 
<img width="459" alt="Screenshot 2024-10-21 at 2 45 25 PM" src="https://github.com/user-attachments/assets/e90fa3bd-7c19-4377-a662-c30b29110cad">